### DISCUSSION TITLE
Compatibility for Jadens Nether Expansion

### DISCUSSION THREAD

**Discussion by SeparateBeast68** [DynamicTreesTeam/DynamicTrees#994] [Compatibility Addon Suggestions]
Jaden's Nether Expansion doesn't add any new trees yet, the mod just tweaks nether forest generation so I don't think making them compatible would be too hard. Now with these mods combined most trees work properly but there is a couple that don't.

https://www.curseforge.com/minecraft/mc-mods/jadens-nether-expansion

**Comment by Groupix05** [comment]
There is just the Warped forest ?

**Comment by Groupix05** [comment]
https://www.curseforge.com/minecraft/mc-mods/dynamic-trees-jadens

**Comment by SeparateBeast68** [comment]
Im pretty sure it also tweaks crimson forests but i will test the compat mod and let you know

**Reply by Groupix05** [reply]
When testing, without dtjadens, crimson forest was not changed. There are blocks for it, but it's not used.

**Comment by supermassimo** [comment]
Done https://www.curseforge.com/minecraft/mc-mods/dynamic-trees-jadens
